.github/workflows/build-model.yml

```diff
@@ -6,10 +6,10 @@ on:
     inputs:
       data_model_version:
         type: string
-        description: "data_model_version: The data standard version corresponding to the Ed-Fi Model branch. Default: 5.0"
+        description: "data_model_version: The data standard version corresponding to the Ed-Fi Model branch. Default: 5.0.0"
```

Comment:
These changes are to make the model build with the latest technology, this file can be ignored